deepchecks/vision/checks/data_integrity/abstract_property_outliers.py

```diff
@@ -60,48 +61,39 @@ class AbstractPropertyOutliers(SingleDatasetCheck):
     """
 
     def __init__(self,
-                 properties: t.List[t.Dict[str, t.Any]] = None,
+                 properties_list: t.List[t.Dict[str, t.Any]] = None,
+                 property_input_type: PropertiesInputType = PropertiesInputType.OTHER,
```

Comment:
Maybe don't put a default value / comment that this must be replaced in the inheriting checks?

Reply:
@noamzbr why?